swift/ql/src/change-notes/2023-11-06-command-injection.md

```diff
@@ -0,0 +1,5 @@
+---
+category: newQuery
+---
+
+* Added new query "System command built from user-controlled sources" (`swift/command-line-injection`) for Swift. This query detects system commands built from user-controlled sources without sufficient validation. The query was previously contributed to the 'experimental' directory but will now run by default for all code scanning users.
```

Comment:
FYI, we've previously called out the contributor in the change note when it's been created by an external contributor. I think we should do the same here.

Reply:
Good idea.  Done.